src/refract/VisitorUtils.h

```diff
@@ -125,6 +125,32 @@ namespace refract
         }
     };
 
+    struct GetEnumValue {
```

Comment:
I think you can. But I will let @klokane answer this. Otherwise the PR looks good.